starter-code/src/SpecificationsOverview.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from "react";
+
+import "./SpecificationsOverview.css";
+
+class SpecificationsOverview extends Component {
+  render() {
+    return (
+      <section className="SpecificationsOverview">
+        <div>
+          <img src="../images/icon1.png" />
+          <h2>Declarative</h2>
+          <p>React makes it painless to create interactive UIs.</p>
+        </div>  
```

Comment:
These "card" divs would be best suited as components, since they are all exactly the same except for their content.   I would make a `Spec` component that takes in `image`, `title` and `subtitle` as props.  I would say these cards need to be components more so than the header for example.  Lets say you have an array of data objects that represent the content of these `Spec` cards.  You could loop over the array of objects and assign each object as a prop to your `Spec` component and achieve this in a few lines.